base/functors.jl

```diff
@@ -67,6 +67,9 @@ call(::LDivFun, x, y) = x \ y
 immutable IDivFun <: Func{2} end
 call(::IDivFun, x, y) = div(x, y)
 
+immutable DotIDivFun <: Func{2} end
+call(::IDivFun, x, y) = x .÷ y
```

Comment:
I followed `DotRemFun`'s implementation. Should that also be changed to `DotRemFun`?